### ISSUE TITLE
Speed and accuracy test

### ISSUE THREAD

**Issue by tanjiarui** [qijiezhao/M2Det#6]
I got one question about the test result after a scrutiny of this paper. Did you train these comparison models based on pytorch? If so, have you tuned hyperparameters in accord with official documents?
Thanks

**Comment by qijiezhao** [comment]
> I got one question about the test result after a scrutiny of this paper. Did you train these comparison models based on pytorch? If so, have you tuned hyperparameters in accord with official documents?
> Thanks

Detailedly，ResNet101-SSD is re-trained and tested, cornernet and refinedet are tested based on their official provided models.
Calculating the ACC is with no controversy(mmAP), while computing the speed always exists divergence. 
At the same time when we release the M2Det code, we will give a new speed comparison chart under fair conditions.
